### PR TITLE
#4 Create new arguments

### PR DESCRIPTION
**Pull request by Monstarrrr**
- **Added post-new-argument form; Added API base for posting new argument (untested)**
- **submitButtonLabel fix**
- **Add arguments & fetch them**

### CODE REVIEW

File: backend/core/models.py [hunk 1]
```diff
@@ -31,31 +39,38 @@ class Posts(models.Model):
...
-    commentCount: models.IntegerField = models.IntegerField(default=0, null=True)
-    favoriteCount: models.IntegerField = models.IntegerField(default=0, null=True)
-    closedDate: models.DateField = models.DateField(null=True)
-    communityOwnedDate: models.DateField = models.DateField(null=True)
+    ownerUserId: models.IntegerField = models.IntegerField(null=True)
+    created_at: models.DateTimeField = models.DateTimeField(auto_now_add=True)
+    updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
+    # MVP moment:
+    # acceptedAnswerId: models.IntegerField = models.IntegerField(null=True)
+    # parentId: models.IntegerField = models.IntegerField(null=True)
```
Comment (seporterfield) [inline backend/core/models.py:52]: we need parentId

**Review by seporterfield — APPROVED** [review]

**Review by seporterfield — APPROVED** [review]

**Comment by Monstarrrr** [comment]
How was it merged ?

**Comment by seporterfield** [comment]
Didn't realize either
It seems it's as long as someone else requested your review, no matter what changes you make you can approve and merge.

**Comment by Monstarrrr** [comment]
> Didn't realize either
> It seems it's as long as someone else requested your review, no matter what changes you make you can approve and merge.

Good job, github

**Pull request merged**